### ISSUE TITLE
Might mention that 5V is not needed for read-only

### ISSUE THREAD

**Issue by arpruss** [brownan/Gamecube-N64-Controller#4]
You might want to make it clear in the documentation that if you don't want to use the rumble, you don't need to connect the 5V pin on the controller.

**Comment by brownan** [comment]
good point! The controller circuits only need the 3.3 volts as I recall

**Comment by arpruss** [comment]
Right: I just checked this, at least on my off-brand controllers.
